src/modelfree/common/generate_activations.py

```diff
@@ -35,7 +36,7 @@ def generate_activations(_run, out_dir, score_configs, score_update,
     activation_dirs = run_external(score_configs, post_named_configs=['save_activations'],
                                    config_updates=score_update, adversary_path=adversary_path)
 
-    def path_generator(trial_root, env_name, victim_index, victim_type, victim_path,
+    def path_generator(trial_root, cfg, env_name, victim_index, victim_type, victim_path,
```

Comment:
Minor point, but I think I'd find the code between here and extract_data easier to read if this were named something like 'activations_path_generator', since it would distinguish between the general callable type of path_generator expected within extract_data and the specific instance of such a thing here.

Reply:
Agree this seems clearer. Also realized it doesn't need to be a closure. Renamed + moved to module level.